src/components/VideoCard.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
```

Comment:
VideoCard should be in its own folder probably, since it has an associated Stencil. Or namespace the React components (same with PageHero)